src/main/java/org/candlepin/config/ConfigProperties.java

```diff
@@ -439,6 +445,7 @@ public static String jobConfig(String jobKey, String cfgName) {
 
             this.put(IDENTITY_CERT_YEAR_ADDENDUM, "5");
             this.put(IDENTITY_CERT_EXPIRY_THRESHOLD, "90");
+            this.put(SCA_X509_CERT_EXPIRY_THRESHOLD, "90");
```

Comment:
I think 90 days is too much for the SCA cert. The ID cert has 90 because it is way more critical to make sure to regenerate it before it gets expired, because if it does gets expired then subman has no way to communicate with candlepin any more, and the user would be forced to unregister/re-register the system.
SCA certs expiring is not that big of a problem, since they would get a new cert the next time they check in anyway. I would say something like 10 days is enough

Reply:
Sure

Reply:
Updated to 10 days